fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/domain/LoanRepositoryWrapper.java

```diff
@@ -44,6 +44,7 @@ public LoanRepositoryWrapper(final LoanRepository repository) {
         this.repository = repository;
     }
 
+    @Transactional
```

Comment:
Missing: @Transactional(readOnly = true)

Reply:
@taskain7 ... could please fix this one remaining read-only transaction? Then we can tick this one of and merge. Thanks again for this contribution.